microservices/s3_to_redshift/s3_to_redshift.py

```diff
@@ -139,9 +138,24 @@ def copy_query(dbtable, batchfile, log):
 
         obj = client.get_object(Bucket=bucket, Key=object_summary.key)
         body = obj['Body']
+        csv_string = ''
```

Comment:
Please add a comment to explain what this block does for future maintainers.

Reply:
Added comment explaining the declaration